main.tf

```diff
@@ -39,3 +40,10 @@ resource "azurerm_servicebus_queue_authorization_rule" "listen_auth_rule" {
 
   listen = true
 }
+
+resource "azurerm_servicebus_subscription_rule" "BBA3-filter" {
+  name            = local.filter_rule_name
+  subscription_id = azurerm_servicebus_subscription.servicebus_subscription.id
+  filter_type     = "SqlFilter"
```

Comment:
If possible this should be configurable (for SSCS we're going to be using a Correlation filter)